ui/src/App.scss

```diff
@@ -2,3 +2,41 @@
 @import "node_modules/nhsuk-frontend/packages/components/button/button";
 @import "node_modules/nhsuk-frontend/packages/components/input/input";
 @import "node_modules/nhsuk-frontend/packages/components/tables/tables";
+
+.spinner-white {
+    border: nhsuk-spacing(1) solid rgba($color_nhsuk-white, 0.3);
+    border-top: nhsuk-spacing(1) solid $color_nhsuk-white;
+    animation: spin 1s linear infinite;
+}
+
+.spinner-blue {
+    animation: spin 2s linear infinite;
+    border: nhsuk-spacing(3) solid $color_nhsuk-grey-4;
+    border-top: nhsuk-spacing(3) solid tint($color_nhsuk-blue, 15%);
+    display: inline-block;
+}
+
+.nhsuk-loader {
+    @include nhsuk-responsive-margin(4, bottom);
+    @include nhsuk-responsive-margin(4, top);
+    position: relative;
+    text-align: center;
+    padding-top: 150px;
+    padding-bottom: 150px;
+}
+
+.nhsuk-loader__text {
```

Comment:
The format we are looking at here is called BEM (Block Element Modifier). It can be used in SCSS in a way that allows style nesting. I'd reccomend moving the 'nhsuk-loader__text' inside 'nhsuk-loader' like so:

```
.nhsuk-loader {
    ...
    &__text {
       ...
    }

}
```

The ampersand allows us to concatenate nested css rules that share a unique block such as anything that has 'loader' in its CSS class.